dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -184,6 +184,10 @@
     @BatchSize(size = 25)
     private SortedSet<FileFormat> outputFileFormats = new TreeSet<>();
 
+    @OneToMany(fetch = FetchType.EAGER, orphanRemoval = true, cascade = CascadeType.ALL)
```

Comment:
Start lazy by default, we've had some pain converting things to be lazy after hitting performance problems.

Reply:
Using lazy breaks tests, not sure how to get it working.
```
failed to lazily initialize a collection of role: io.dockstore.webservice.core.Version.authors, could not initialize proxy - no Session
```

Reply:
ya, i don't see how this could easily be changed to actually lazy loading when the frontend depends on it being eager

Reply:
Maybe I misunderstand, but it seems that Version.authors is new, what the UI is depending on since there is no accompanying UI PR is probably Version.getAuthor below which currently means that when the Java object is serialised to JSON it will have to lazily load authors in order to do `authors.stream().findFirst()`

That said, if we had code that was using Version but not serialising it, for example if we needed to look through the versions of a workflow to calculate something about fileformats for example, this being eager would eagerly load authors from the database regardless of whether it was needed or not. 

Additionally, even if the UI immediately needed authors rather that getAuthor, validations below is lazily loaded but seems to serialise fine with calls like https://dockstore.org/api/workflows/path/workflow/github.com%2FDataBiosphere%2Ftopmed-workflows%2Fchecker_UM_variant_caller_wdl/published?include=validations&services=false&versionName=1.32.0

Reply:
`fetch = FetchType.LAZY` is more of a suggestion for hibernate. From what I've seen, the mere existence of getAuthor() (regardless of serialization) will force it to eager load in most places.

Though the failed tests when swapped to `fetch = FetchType.LAZY` suggests that it's not forcing it to eager as much as I thought... :shrug: